R/mssql.R

```diff
@@ -6,6 +6,7 @@ mssql_sys_db <- function(con, dbname, name, vars = NULL) {
     fq_name <- paste0(dbname, ".", name)
     sql_name <- dbname
   }
+  browser()
```

Comment:
I didn't do any changes to MS SQL as I don't have a database to test this with. I left this `browser()` as a reminder 😄